apps/src/dropletUtils.js

```diff
@@ -404,7 +417,15 @@ exports.generateAceApiCompleter = function (functionFilter, dropletConfig) {
         callback(null, []);
         return;
       }
-      callback(null, apis);
+      var line = session.getLine(pos.row);
+      var identifier = acUtil.retrievePrecedingIdentifier(line, pos.column);
+      if (pos.column > (identifier.length + 1) &&
+          line[pos.column - identifier.length - 1] === '.') {
```

Comment:
Since we're not actually examining the thing before the dot yet, I'd sort of like to use the methods completer even if the line begins with a dot, to support the multiline chaining convention:

```javascript
var x = makeAThing(y)
.andChangeIt()
.andConfigureIt();
```

That's the most likely scenario I can think of where a line would begin with a dot, but I'm sure it's not the only one.  On the other hand if you feel like this takes us down the wrong path (given we'd like to examine the LHS of the dot eventually) I'm fine with leaving as-is.